Realm/Tests/Swift/SwiftPropertyTypeTest.swift

```diff
@@ -121,6 +121,25 @@ class SwiftPropertyTypeTest: RLMTestCase {
         }
         XCTAssertNotNil(succeeded, "Writing an object with an ignored lazy property should work.")
     }
+
+    func testObjectiveCTypeProperties() {
+        let realm = realmWithTestPath()
+        var object: SwiftObjectiveCTypesObject!
+        let now = NSDate()
+        let data = "fizzbuzz".data(using: .utf8)! as Data as NSData
```

Comment:
Swift bug. Should I add a FIXME everywhere we have to workaround this? https://bugs.swift.org/browse/SR-2365